src/lib/utils.py

```diff
@@ -70,11 +70,16 @@ def _extract_timestamp(info):
 
 
 def strip_query(url):
-    """Strips the query string from the given URL, if any."""
+    """Sanitizes the query string from the given URL, if any. Parameters whose
+    names begin with an underscore are assumed to be tracking identifiers and]
```

Comment:
Stray typo I assume.
```suggestion
    names begin with an underscore are assumed to be tracking identifiers and
```

Reply:
Thanks!